packages/generator/generators/react-cmf/templates/src/app/configure.js

```diff
@@ -10,10 +10,4 @@ export default {
 	initialize() {
 		registerComponent('App', App);
 	},
-	getStore() {
```

Comment:
Does it mean I cannot access store for dispatching outside of components with connect()?